### ISSUE TITLE
[BUG]: llama2 examples finetune.py loss is always nan , when i training

### ISSUE THREAD

**Issue by LXXiaogege** [hpcaitech/ColossalAI#4855]
### 🐛 Describe the bug

When using the finetune.py script to stf my dataset, the loss is always nan from start to finish, and the pre-training model I'm using is llama2-chinese-7b-chat, not the colossal-llama2 

### Environment

My environment is python=3.8 torch=1.13.1, cuda=11.7, colossalai=0.3.3

**Comment by LXXiaogege** [comment]
I replaced the llama2-chinese-7b-chat model with colossal-llama2 this problem is fixed, should be not support other versions of llama2
